roles/openshift_storage_glusterfs/tasks/restart_node.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
The name of this file is not great.

Should be recreate_pods.yml; restart_node.yml implies we're rebooting the host, or possibly restarting the kubelet.